src/recensio/plone/profiles/default/types/Review_Journal.xml

```diff
@@ -47,7 +47,6 @@
     <element value="plone.ownership" />
     <element value="plone.publication" />
     <element value="plone.shortname" />
-    <element value="recensio.authors" />
```

Comment:
Why are you removing the authors behavior here? Review Journal needs it, see https://github.com/syslabcom/scrum/issues/884

Reply:
Oh, I see that the behavior is not listed under the heading “Review Journal” in the ticket. That's probably my mistake. I'll add it there.

Reply:
Because I recheked the list and saw:
```
'authors(reference:rw)': 'ReviewMonograph', 'ReviewArticleCollection', 'ReviewArticleJournal'
```
I will drop this commit...

Reply:
Dammit, no. It's Review Article Journal that needs the behavior, not Review Journal. It's fine to remove it here. Sorry for the noise.